ksqldb-common/src/main/java/io/confluent/ksql/schema/ksql/types/SqlTypes.java

```diff
@@ -27,6 +27,7 @@ private SqlTypes() {
   public static final SqlPrimitiveType BIGINT = SqlPrimitiveType.of(SqlBaseType.BIGINT);
   public static final SqlPrimitiveType DOUBLE = SqlPrimitiveType.of(SqlBaseType.DOUBLE);
   public static final SqlPrimitiveType STRING = SqlPrimitiveType.of(SqlBaseType.STRING);
+  public static final SqlPrimitiveType DECIMAL = SqlPrimitiveType.of(SqlBaseType.DECIMAL);
```

Comment:
This should be removed. Instead use the ‘decimal’ builder method in the line below to build decimals.

Reply:
Precision and scale aren't available in `SchemaConverters.FunctionToSql.toSqlType()` since the function only takes a ParamType currently. I think we'd have to enhance the DecimalType to return the precision and value in order to use the builder method properly.